doc/Changelog.md

```diff
@@ -0,0 +1,20 @@
+Changelog {#Changelog}
+=========
+
+# Release 1.1.1 {#Release010101}
+
+* Build fixes for gcc 4.9.2 and MSCV.
+
+# Release 1.1 {#Release010100}
+
+* Added the function URI::getAuthority
+* Added some URI setters.
+
+# Release 1.0.1 {#Release010010}
+
+* Fixed URI parsing for URI without authority.
+
+# Release 1.0 {#Release010000}
```

Comment:
Dates and tags can only be approximate unless I collapse 1.1 and 1.1.1 into a single one. The problem is that I made a mistake when bumping from 1.0.1 to 1.1 because I forgot to reset the patch number to 0. So in the history, version 1.1.0 doesn't really exist. The best solution I can think of is omitting 1.1.